--- jwm-2.0.1/src/jwm.h.orig	2010-09-13 00:50:26.183999850 +0200
+++ jwm-2.0.1/src/jwm.h	2010-09-13 00:51:05.381999851 +0200
@@ -88,7 +88,8 @@
 #   endif
 #   ifdef USE_FRIBIDI
 #      include <fribidi/fribidi.h>
-#      include <fribidi/fribidi_char_sets_utf8.h>
+#      include <fribidi/fribidi-char-sets-list.h>
+#      include <fribidi/fribidi-char-sets.h>
 #   endif
 
 #endif /* MAKE_DEPEND */
